keps/sig-storage/1432-volume-health-monitor/README.md

```diff
@@ -674,8 +679,200 @@ _This section must be completed when targeting alpha to a release._
   disable this feature is to install or unistall the sidecars, we cannot write
   tests for feature enablement/disablement.
 
+### Rollout, Upgrade and Rollback Planning
+
+_This section must be completed when targeting beta graduation to a release._
+
+* **How can a rollout fail? Can it impact already running workloads?**
+  Try to be as paranoid as possible - e.g., what if some components will restart
+   mid-rollout?
+   This feature does not have a feature gate. It is enabled when the health
+   monitoring controller and agent sidecars are deployed with the CSI driver.
+   So the only way for a rollout to fail is that deploying the health
+   monitoring controller or agent sidecars with the CSI driver fails. If
```

Comment:
Because these are deployed as sidecars, if the sidecars fail, that would also result in a restart of the rest of the CSI driver, which certainly could impact running workloads, right? How are we mitigating that risk---is it up to cluster operators to add and remove these sidecars from their cluster manifests?